olp-cpp-sdk-core/src/http/ios/OLPHttpTask.mm

```diff
@@ -101,8 +101,9 @@ - (OLPHttpTaskStatus)run {
   // Cache the task id for fast retrieve later on
   [_httpClient registerDataTask:_dataTask forHttpTask:self];
 
-  OLP_SDK_LOG_TRACE_F(
-      kLogTag, "Run task: id=[%llu]; task_id=[%lu];\t httpMethod=[%s];\t [%s])",
+  OLP_SDK_LOG_DEBUG_F(
+      kLogTag,
+      "Run task, request_id=%llu, task_id=%lu,\t http_method=%s,\t url=%s)",
```

Comment:
I strongly advise against multiline comments. These look nice on the command line, but if you need to get it from remote or some log file these will look like crap.